### DISCUSSION TITLE
Default disable or prompt running github actions on forks

### DISCUSSION THREAD

**Discussion by FireLemons** [community/community#7377] [Actions]
I have a fork of a repo and just today realized it was trying to run the same workflows as the parent repo's `.github` folder. I didn't need to run any of the workflows. Maybe it would save on resources to disable all the workflows in a fork by default but also warn the fork owner. Or forking could prompt the fork owner if they would like to enable running actions.

**Comment by xgdgsc** [comment]
Yes. For some big repos it could generate 100+ emails per day. Like when you fork [julia](https://github.com/JuliaLang/julia)
